# HG changeset patch
# User Kai T. Ohlhus <k.ohlhus@gmail.com>
# Date 1488022067 -3600
#      Sat Feb 25 12:27:47 2017 +0100
# Node ID eb6059699e3b178c3d99a96caf8eab1eb8636ca0
# Parent  ac09d0a6c6a09b465869152f54ea48b7d913a1f1
doc: add missing contributors from hg log to contributors.in (bug #46740)

* doc/interpreter/contributors.in: add missing contributors from hg log.

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -1,30 +1,34 @@
 Ben Abbott
 Drew Abbot
+NVS Abhilash
 Andy Adler
 Adam H. Aitkenhead
 Joakim Andén
 Giles Anderson
 Joel Andersson
 Lachlan Andrew
 Pedro Angelo
 Damjan Angelovski
 Muthiah Annamalai
 Markus Appel
 Branden Archer
 Willem Atsma
 Marco Atzeri
 Ander Aurrekoetxea
 Shai Ayal
+Sahil Badyal
+Jeff Bai
 Roger Banks
 Ben Barrowes
 Alexander Barth
 David Bateman
 Heinz Bauschke
+Miguel Bazdresch
 Julien Bect
 Stefan Beller
 Roman Belov
 Markus Bergholz
 Karl Berry
 Atri Bhattacharya
 Ethan Biery
 David Billinghurst
@@ -43,22 +47,26 @@ Marco Caliari
 Daniel Calvelo
 John C. Campbell
 Juan Pablo Carbajal
 Jean-Francois Cardoso
 Joao Cardoso
 Larrie Carr
 David Castelow
 Vincent Cautaerts
+Marco Cecchetti
+Corbin Champion
 Clinton Chee
 Albert Chin-A-Young
+Sunghyun Cho
 Carsten Clark
 Catalin Codreanu
 J. D. Cole
 Jacopo Corno
+Andre da Costa Barros
 Martin Costabel
 Michael Creel
 Richard Crozier
 Jeff Cunningham
 Martin Dalecki
 Jacob Dawid
 Jorge Barros de Abreu
 Carlo de Falco
@@ -67,82 +75,91 @@ Philippe Defert
 Bill Denney
 Fabian Deutsch
 Christos Dimitrakakis
 Pantxo Diribarne
 Vivek Dogra
 John Donoghue
 David M. Doolin
 Carnë Draug
+Sergey Dudoladov
 Pascal A. Dupuis
 John W. Eaton
 Dirk Eddelbuettel
 Pieter Eendebak
 Paul Eggert
 Stephen Eglen
 Peter Ekberg
 Garrett Euler
 Edmund Grimley Evans
 Rolf Fabian
+Francesco Faccio
 Gunnar Farnebäck
 Massimiliano Fasi
 Stephen Fegan
 Ramon Garcia Fernandez
 Torsten Finke
+David Finkel
 Colin Foster
 Jose Daniel Munoz Frias
 Brad Froehle
 Castor Fu
 Eduardo Gallestey
 Walter Gautschi
 Klaus Gebhardt
 Driss Ghaddab
 Eugenio Gianniti
 Hartmut Gimpel
+Michele Ginesi
 Nicolo Giorgetti
 Arun Giridhar
 Michael D. Godfrey
 Michael Goffioul
 Glenn Golden
 Tomislav Goles
 Keith Goodman
 Brian Gough
+Alexander Graf
 Michael C. Grant
 Steffen Groot
 Etienne Grossmann
 David Grundberg
 Kyle Guinn
 Vaibhav Gupta
 Peter Gustafson
 Kai Habel
 Patrick Häcker
 William P. Y. Hadisoeseno
 Jaroslav Hajek
 Benjamin Hall
+Alexander Hansen
 Kim Hansen
 Søren Hauberg
 Dave Hawthorne
 Oliver Heimlich
 Daniel Heiserer
 Piotr Held
 Martin Helm
 Stefan Hepp
 Martin Hepperle
 Jordi Gutiérrez Hermoso
+Israel Herraiz
 Yozo Hida
 Ryan Hinton
 Roman Hodek
 A. Scottedward Hodel
 Richard Allan Holcombe
 Tom Holroyd
 David Hoover
 Kurt Hornik
+Craig Hudson
 Christopher Hulbert
 Cyril Humbert
 John Hunt
+Stefan Husmann
 Teemu Ikonen
 Alan W. Irwin
 Allan Jacobs
 Geoff Jacobsen
 Vytautas Jančauskas
 Nicholas R. Jankowski
 Mats Jansson
 Robert Jenssen
@@ -164,25 +181,27 @@ Lars Kindermann
 Aaron A. King
 Erik Kjellson
 Arno J. Klaassen
 Alexander Klein
 Geoffrey Knauth
 Heine Kolltveit
 Ken Kouno
 Kacper Kowalik
+Endre Kozma
 Daniel Kraft
 Nir Krakauer
 Aravindh Krishnamoorthy
 Oyvind Kristiansen
 Artem Krosheninnikov
 Piotr Krzyzanowski
 Volker Kuhlmann
 Ilya Kurdyukov
 Tetsuro Kurita
+Ben Kurtz
 Philipp Kutin
 Miroslaw Kwasniak
 Rafael Laboissiere
 Kai Labusch
 Claude Lacoursiere
 Walter Landry
 Bill Lash
 Dirk Laurie
@@ -224,21 +243,23 @@ Thorsten Meyer
 Stefan Miereis
 Petr Mikulik
 Mike Miller
 Serviscope Minor
 Stefan Monnier
 Rafael Monteiro
 Antoine Moreau
 Kai P. Mueller
+Amod Mulay
 Armin Müller
 Hannes Müller
 Victor Munoz
 PrasannaKumar Muralidharan
 Iain Murray
+Nicholas Musolino
 Markus Mützel
 Carmen Navarrete
 Todd Neal
 Philip Nienhuis
 Al Niessner
 Felipe G. Nievinski
 Rick Niles
 Takuji Nishimura
@@ -274,33 +295,36 @@ Hans Ekkehard Plesser
 Sergey Plotnikov
 Tom Poage
 Nathan Podlich
 Orion Poplawski
 Ondrej Popp
 Jef Poskanzer
 Francesco Potortì
 Konstantinos Poulios
+Kostas Poulios
 Jarno Rajahalme
 Eduardo Ramos
 Pooja Rao
 James B. Rawlings
 Eric S. Raymond
 Balint Reczey
 Joshua Redstone
+Andy Register
 Lukas Reichlin
 Michael Reifenberger
 Ernst Reissner
 Jens Restemeier
 Anthony Richardson
 Jason Riedy
 E. Joshua Rigler
 Sander van Rijn
 Petter Risholm
 Matthew W. Roberts
+Melvin Robinson
 Dmitry Roshchin
 Peter Rosin
 Andrew Ross
 Fabio Rossi
 Mark van Rossum
 Joe Rothweiler
 David Rörich
 Kevin Ruland
@@ -330,24 +354,27 @@ Ahsan Ali Shahid
 Baylis Shanks
 Andriy Shinkarchuck
 Robert T. Short
 Joseph P. Skudlarek
 John Smith
 Julius Smith
 Shan G. Smith
 Peter L. Sondergaard
+Rüdiger Sonderfeld
 Joerg Specht
 Quentin H. Spencer
 Christoph Spiel
 David Spies
 Richard Stallman
 Russell Standish
+Ryan Starret
 Brett Stewart
 Doug Stewart
+Jen Stewart
 Jonathan Stickel
 Judd Storrs
 Thomas Stuart
 Bernardo Sulzbach
 Ivan Sutoris
 John Swensen
 Daisuke Takago
 Ariel Tankus
@@ -356,20 +383,22 @@ Duncan Temple Lang
 Matthew Tenny
 Kris Thielemans
 Georg Thimm
 Corey Thomasson
 Andrew Thornton
 Olaf Till
 Christophe Tournery
 Thomas Treichl
+Abhinav Tripathi
 Karsten Trulsen
 David Turner
 Frederick Umminger
 Utkarsh Upadhyay
+José Vallet
 Stefan van der Walt
 Peter Van Wieren
 James R. Van Zandt
 Risto Vanhanen
 Gregory Vanuxem
 Mihas Varantsou
 Ivana Varekova
 Sébastien Villemot
@@ -385,12 +414,14 @@ Andreas Weingessel
 Martin Weiser
 Michael Weitzel
 David Wells
 Joachim Wiesemann
 Georg Wiora
 Fook Fah Yap
 Sean Young
 Michele Zaffalon
+Serhiy Zahoriya
 Johannes Zarl
 Michael Zeising
 Federico Zenith
+Claudius Zingerli
 Alex Zvoleff
